Model/Method/PayPerEmail.php

```diff
@@ -139,6 +139,11 @@ public function getOrderTransactionBuilder($payment)
         $services = [];
         $services[] = $this->getPayperemailService($payment);
 
+        $cmService = $this->serviceParameters->getCreateCombinedInvoice($payment, 'payperemail');
```

Comment:
$this->buckarooPaymentMethodCode may be used

Reply:
yes, thanks, can be used,  fixed